chapters/equations.tex

```diff
@@ -555,7 +555,7 @@ \section{Synchronous Data-flow Principle and Single Assignment Rule}\label{synch
 Modelica is based on the synchronous data flow principle and the single
 assignment rule, which are defined in the following way:
 \begin{enumerate}
-\item Discrete variables keep their actual values until these variables are explicitly changed.
+\item Discrete-time variables keep their actual values until these variables are explicitly changed.
```

Comment:
Shouldn't this be _Variables assigned in when-clauses_?

Reply:
No, the previous statement also applied to other Integer/Boolean variables and I didn't want to change that.
```
 Integer i;
algorithm 
 if sin(time)>0.9 then
   i:=1;
 end if;
 if sin(time)<-0.9 then
   i:=-1;
 end if;
```
As far as I recall there are models relying on this. One could make some argument that this is slightly different; (the algorithm implicitly sets the variable to its pre) but that seems contrived.

Reply:
Sure.  Resolving.